SupportFiles/jira_plugin-install.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+# shellcheck disable=SC2015
+#
+# Script to download and install JIRA Plugins from other sources
+#
+#################################################################
+# shellcheck disable=SC2086
+PROGNAME="$(basename ${0})"
+SCRIPTHOME="${HOME:-/root}"
+
+JIRADCHOME="/var/atlassian"
+JIRAINSTDIR="/opt/atlassian/jira"
+DBCFGDIR="${JIRADCHOME}/application-data/jira"
+PLUGINS1DIR="${JIRAINSTDIR}/atlassian-jira/WEB-INF/lib"
+PLUGINS2TMP="$SCRIPTHOME/plugins2"
+PLUGINS2DIR="${DBCFGDIR}/plugins"
+INSTLPLUGINS2DIR="${DBCFGDIR}/plugins/installed-plugins"
+
+# Array of URLs to download plugins (Type 1 or 2)
+PLUGINS1JARURL=(
+                 # "<http or https link to Type 1 jar file>"
+               )
+PLUGINS2JARURL=(
+                 # "<http or https link to Type 2 jar or obr file>"
+               )
+
+# Misc error-handler
+function err_exit {
+   local ERRSTR="${1}"
+   local SCRIPTEXIT=${2:-1}
+
+   # Our output channels
+   # echo "${ERRSTR}" > /dev/stderr
+   logger -t "${PROGNAME}" -p kern.crit "${ERRSTR}"
+
+   # Need our exit to be an integer
+   if [[ ${SCRIPTEXIT} =~ ^[0-9]+$ ]]
+   then
+      exit "${SCRIPTEXIT}"
+   else
+      exit 1
+   fi
+}
+
+# Make git run quietly...
```

Comment:
This function appears unnecessary: not performing any git fetch/clone operations elsewhere in the script?

Reply:
I agree.  I'll remove it.  Leftover from the base script I leveraged.